use_cases/face_mask_detection.yml

```
@@ -1,5 +1,5 @@
 nodes:
-- input.live
+- input.visual
```

Same point as the .rsts: demo intended to be run using webcam to interact with use case better.
```suggestion
   - input.visual:
       source: 0
```
Same for group size checking, object_counting_present, privacy_protection_faces, social_distancing, zone_counting.